README.md

```diff
@@ -254,15 +254,17 @@ class IterableWeakMap {
     }
 
     this.#weakMap.delete(key);
-    this.#refMap.delete(entry.ref);
+    this.#refSet.delete(entry.ref);
     this.#finalizationGroup.unregister(entry.ref);
     return true;
   }
 
   *[Symbol.iterator]() {
-    for (const [ref, value] of this.#refMap) {
+    for (const ref of this.#refSet) {
       const key = ref.deref();
-      if (key) yield [key, value];
+      if (!key) continue;
+      const { value } = this.#refMap.get(key);
```

Comment:
I believe this needs to be this.#weakMap, not this.#refMap

Reply:
Oops, sorry for landing too soon. Could someone make a follow-up PR?